src/Person/Person.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './Person.scss';
+
+function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName: pName,
+}) {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{age && `I am ${age}`}</p>}
```

Comment:
Ти вже зробив цю перевірку, не потрібно робити її двічі
```suggestion
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```

Reply:
Прокліпав це